libs/@blockprotocol/type-system/package.json

```diff
@@ -69,6 +69,7 @@
     "rollup": "3.5.1",
     "ts-jest": "29.0.5",
     "tslib": "2.4.1",
-    "typescript": "4.9.4"
+    "typescript": "4.9.4",
+    "wasm-pack": "0.10.3"
```

Comment:
Have you removed your local setup and tested this quite a bit?

I remember this was one of the first approaches we tried and it had similar problems on M1 and such. (I also just really don't like the idea of loading a rust binary through `yarn` but _if_ it works I'm fine with it)

Also who maintains this distribution? Who are we dependent on updates from

Reply:
Yes, everything is tested locally. `which wasm-pack` points to `.node_modules`.

The package is maintained by https://github.com/rustwasm

Reply:
Hm, let's try it out and see if there are issues then :)